src/index.ts

```diff
@@ -71,6 +71,20 @@ export interface ProjectBuildOptions {
   js?: {minify?: boolean, compile?: boolean};
 }
 
+export interface LintOptions {
+  /**
+   * The lint rules to run. Can be the code of a collection of rules like
+   * "polymer-2.0" or an individual rule like "dom-module-invalid-attrs".
```

Comment:
Let's refer to Polymer versions as `polymer-1` and `polymer-2`, since we'll only have 2.0 be the latest for a very short time.